File: CPP/struct_student_details.cpp
//Structure for Student (input details of one student)
#include <iostream>
using namespace std;

struct Student {
    string name;
    int roll;
    float marks;
};

int main() {
    Student s;

    cout << "Enter name: ";
    cin >> s.name;
    cout << "Enter roll number: ";
    cin >> s.roll;
    cout << "Enter marks: ";
    cin >> s.marks;

    cout << "\nStudent Details:\n";
    cout << "Name: " << s.name << "\nRoll: " << s.roll << "\nMarks: " << s.marks << endl;

    return 0;
}

File: Python/struct_student_details.py
#Class for Student (input details of one student)
class Student:
    def __init__(self, name, roll, marks):
        self.name = name
        self.roll = roll
        self.marks = marks

# Input
name = input("Enter name: ")
roll = int(input("Enter roll number: "))
marks = float(input("Enter marks: "))

s = Student(name, roll, marks)

print("\nStudent Details:")
print("Name:", s.name)
print("Roll:", s.roll)
print("Marks:", s.marks)
